scripts/run-framework/agief_experiment/experiment.py

```diff
@@ -192,16 +192,33 @@ def run_parameterset(self, compute_node, cloud, args, entity_filepath, data_file
             # log results expressed in the appropriate entity config
             self.log_results_config(compute_node)
 
+            # Get the path to labels and features CSV files
+            out_labels_file_path = self.experiment_utils.datapath('labels.csv')
```

Comment:
It is ok to put this in code as we discussed, but move the file names into variables that are easily visible and modifiable. I suggest making them Class constants like `TEMPLATE_PREFIX` but open to other options.